tests/foreman/ui_airgun/test_syncplan.py

```diff
@@ -0,0 +1,119 @@
+"""Test class for Sync Plan UI
+
+:Requirement: Syncplan
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: UI
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+from datetime import datetime, timedelta
+
+from robottelo.datafactory import gen_string
+from robottelo.decorators import tier2
+
+
+def get_client_datetime(browser):
```

Comment:
I have better idea - you could move this helper inside `AirgunBrowser`, so you could use it like
```python
session.browser.get_client_datetime()
```
Or even make a property like `session.browser.current_datetime`
Or, if you don't like previous ideas - you could at least move this helper to `robottelo.helpers`. Test module is definitely not the best place to put such generic helper in.

Reply:
huh, it will never be used in any other place, but fine